packages/main/src/MultiInput.js

```diff
@@ -0,0 +1,188 @@
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import Input from "./Input.js";
+import MultiInputTemplate from "./generated/templates/MultiInputTemplate.lit.js";
+import styles from "./generated/themes/MultiInput.css.js";
+import Token from "./Token.js";
+import Tokenizer from "./Tokenizer.js";
+import Icon from "./Icon.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-multi-input",
+	properties: /** @lends sap.ui.webcomponents.main.MultiInput.prototype */ {
+		/**
+		 * Determines whether a value help icon will be should in the end of the input.
+		 * Pressing the icon will fire <code>value-help-icon-press</code> event.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		showValueHelpIcon: {
+			type: Boolean,
+		},
+
+		/**
+		 * Indicates whether the tokenizer is expanded or collapsed(shows the n more label)
+		 * @private
+		 */
+		expandedTokenizer: {
```

Comment:
should not use the word "tokenizer" at all, this is a technical implementation, the users don't know what this means.

Reply:
This is a private property, any proposal for another name?

Reply:
In UI5 we have introduced the enumeration renderMode with values "Narrow" and "Loose", which we believe describe the "rendering behaviour" well. Also it allows for expansion in future with other modes.